capstone/src/test/javascript/calendarTest.js

```diff
@@ -0,0 +1,20 @@
+describe('DataTable Construction', function() {
+  it('Will add correct columns/rows to DataTable from JSON', function() {
+    const mockCalenderInfo = {};
+    const expectedFreqData;
+    const expectedTimelineData;
+    const actualFreqData;
+    const actualTimelineData;
+    window.initMap = function() {
+      // TODO(ihsan314): adjust table and mock calendar info
+      // when format of JSON is confirmed
+      expectedFreqData = new google.visualization.DataTable();
+      expectedTimelineData = new google.visualization.DataTable();
+      actualFreqData = new google.visualization.DataTable();
+      actualTimelineData = new google.visualization.DataTable();
+      populateDataTables(actualFreqData, actualTimelineData, mockCalenderInfo);
```

Comment:
i don't think any of this code is being invoked. it would make sense to refactor the functions you want to test out of the callback in calendar.js to isolate them for test...

Reply:
Ok I'll look at #59 for isolating functions out of the callback. Additionally I'll merge the master branch again so that I can take advantage of #68.